index.bs

```diff
@@ -600,17 +600,71 @@ describes, as is Internet Explorer's {{window.toStaticHTML()}}.
+    of these are explained here, with an implementation in the [[DEFAULTS]]
+    script. It is expected that these values will change before this
+    specifcation is finalized. Also, we expect these
+    to be updated to include additional HTML elements as they are
     introduced in user agents.
 
+For the purpose of this Sanitizer API, [[HTML]] constructs fall into one of
+four classes, where the first defines the baseline, and the first, second,
+plus the third define the default:
+
+1. Elements and attributes that (directly) execute script.
+   In other words, elements and attributes that are unconditionally script-ish.
+2. Legacy and "difficult" elements and attributes.
+  Examples are the `<plaintext>` `<xmp>` and elements, which have special
+  parsing rules attached to them. These are not dangerous _per se_, but they
+  have contributed to existing vulnerability.
+3. Elements and attributes that we feel rarely make sense in user-supplied
+  content.
+4. All the rest.
+
+Specifically:
+
+1. Script-ish constructs:
+  - The {{HTMLScriptElement}}, which proudly executes script as its sole purpose.
+  - All [event handler attributes]((https://html.spec.whatwg.org/#event-handler-attributes),
+    since these also execute script.
+  - {{HTMLIFrameElement}}, which loads arbitrary HTML content and therefor also script.
+  - The legacy {{HTMLObjectElement}} and {{HTMLEmbedElement}}, which load
+    non-HTML active content. Also, `<object>`'s side-kick {{HTMLParamElement}}.
+  - The [no-longer conforming](https://html.spec.whatwg.org/#non-conforming-features)
+    `<frame>`, `<frameset>`, and `<applet>` tags, which are
+    outdated versions companions of several elements listed above.
+  - The `<noframes>`, `<noembed>`, and `<nolayer>` elements.
+    These, by themselves, are arguably not script-ish, but they are companions
+    to elements listed above, and make no sense on their own.
+  - Also, the {{HTMLBaseElement}}, as this effectively modifies interpretation
+    of other URLs.
+2. Legacy and "difficult" elements.
+  - Special parsing behaviour. This is not dangerous in its own right, but has
+    contributed to mXSS-style attacks. This includes:
+      - `<plaintext>` (Which parses in [=PLAINTEXT state=].)
+      - The non-conforming [`<xmp>`](https://html.spec.whatwg.org/#xmp) element.
+  - Legacy elements:
+      - `<image>` ([which is parsed as `<img>`](https://html.spec.whatwg.org/#parsing-main-inbody)).
+      - `<basefont>`
+
+3. Constructs unlikely to be beneficial in user-supplied content:
+  - The {{HTMLTemplateElement}}, which introduces a new template to be used
+    by JavaScript, and its {{HTMLSlotElement}} accomplice.
+  - The frame-like [HTMLPortalElement](https://wicg.github.io/portals/).
+  - {{HTMLDataElement}},
+  - The (deprecated) [allowpaymentrequest](https://www.w3.org/TR/payment-request/) attribute.
+
+
 ## The Baseline Element Allow List ## {#baseline-element-allow-list}
 
+<em>This subsection is normative.</em>
```

Comment:
do we need those hints for this section (and below)? not opposed, but it surprised me :)

Reply:
No idea, honestly. My thinking was: I can no longer declare the entire section as normative/non-normative because this now differs for sub-sections. Therefore, I'll have to declare it for each subsection now.

I do wonder whether this makes sense, or wether I'm being to "algorithm-y" here. I mean, a reader could certainly figure our that if one sub-section is marked _non-normative_, the presumably the others are normative by implication.

Reply:
So... I went back to this, because I didn't really like this either. Now the appendix says once, "normative, except where noted otherwise", and then has the one sub-section declare itself non-normative. I think that reads well, overall...